substrate/utils/wasm-builder/src/wasm_project.rs

```diff
@@ -868,6 +868,11 @@ fn build_bloaty_blob(
 		// We don't want to call ourselves recursively
 		.env(crate::SKIP_BUILD_ENV, "");
 
+	let cargo_args = &env::var(crate::WASM_BUILD_CARGO_ARGS).unwrap_or_default();
```

Comment:
```suggestion
	let cargo_args = env::var(crate::WASM_BUILD_CARGO_ARGS).unwrap_or_default();
```

Reply:
Done in: [bbcfab1](https://github.com/paritytech/polkadot-sdk/pull/5838/commits/bbcfab144aa85bcb2020c5df9dc5ecf6e7a52ff9)